e2e/__tests__/Attachment.spec.js

```diff
@@ -5,14 +5,15 @@ import fs from 'fs'
 import { spy } from 'sinon'
 import { file } from 'chai-files'
 import rimraf from 'rimraf'
+import noop from 'lodash/noop'
 
 const TEMP_ASSETS_DIRECTORY = `${__dirname}/.tmp`
 
 describe('attachment', () => {
   const veemSDK = new VeemSDK(CONFIG)
 
   beforeAll(async () => {
-    await fs.mkdir(TEMP_ASSETS_DIRECTORY, { recursive: true })
+    await fs.mkdir(TEMP_ASSETS_DIRECTORY, { recursive: true }, noop)
```

Comment:
You could also just do `fs. mkdirSync` here and not provide a cb.

Reply:
That makes more sense 👍